lib/core/search-text/search-text-input.component.ts

```diff
@@ -279,4 +299,19 @@ export class SearchTextInputComponent implements OnInit, OnDestroy {
         this.onDestroy$.next(true);
         this.onDestroy$.complete();
     }
+
+    canShowClearSearch(): boolean {
+        return this.showClearButton && this.isSearchBarActive();
+    }
+
+    onClearSearch() {
+        if (this.defaultState === SearchTextStateEnum.collapsed) {
+            this.searchTerm = '';
+            this.subscriptAnimationState = this.animationStates[this.dir].inactive;
+            this.searchChange.emit('');
```

Comment:
what's the difference between the even reset and clear? let's review this event name